dao/mongo.go

```diff
@@ -300,3 +306,25 @@ func (m *MongoService) GetPaymentsWithRefundPendingStatus() ([]models.PaymentRes
 
 	return payments, nil
 }
+
+func (m *MongoService) PatchPaymentsWithRefundPendingStatus(id string, paymentUpdate *models.PaymentResourceDB) error {
```

Comment:
Function needs a comment on the preceding line

Reply:
Fixed.Thanks